starters/nuxt-2/utils/buildRoutes.js

```diff
@@ -0,0 +1,33 @@
+import { Storefront } from '@nacelle/storefront-sdk';
+
+export const buildRoutes = async () => {
+  const client = new Storefront({
+    storefrontEndpoint: process.env.NACELLE_STOREFRONT_ENDPOINT,
+    token: process.env.NACELLE_STOREFRONT_TOKEN,
+    locale: process.env.NACELLE_STOREFRONT_LOCALE
+  });
+  const QUERY = `
+    {
+      products {
+        content {
+          handle
+        }
+      }
+      productCollections {
+        content {
+          handle
+        }
+      }
+    }
+  `;
+
+  const response = await client.query({ query: QUERY });
+  return [
+    ...response?.products?.map(
+      (product) => `/products/${product.content.handle}`
+    ),
+    ...response?.productCollections?.map(
+      (productCollection) => `/collections/${productCollection.content.handle}`
+    )
```

Comment:
Minor nitpick, but after testing with the Nacelle Storefront API, it looks like the result of these queries should always be `[]`, and not falsey (`null`), so we could lose the `?` before the `.map`s.

<img width="800" alt="productCollections nonexistent" src="https://user-images.githubusercontent.com/5732000/150704963-d198c99b-d1b3-414e-8248-d1a6965b3496.png">